backend/question.py

```diff
@@ -6,3 +6,73 @@
 @question_page.route('/question/ping', methods=['GET'])
 def question_ping():
     return "Pong", 200
+
+# allow user to post a question and word account not more than 200 words
+@question_page.route('/question/add', methods=['POST'])
+def question_add():
+    data = request.get_json()
+    user_id = data['user_id']
```

Comment:
i added get_user_id_from_header in the helper function. If you could use this function? or simply make it
user_id = int(data['user_id'])